Microsoft.Dynamics365.UIAutomation.Sample/UCI/ApiTests/TestsBase_Tests.cs

```diff
@@ -47,5 +45,25 @@ public void NavigationFail_BrowserIsClosed()
                 Assert.IsNull(_client, "Browser still open");
             }
         }
+
```

Comment:
Sample & Tests. Is the same test, but are require two to reproduce the error. I have not better Idea as check as mark as inconclusive & check visually that the printed traces are the same. Maybe somebody else has eine better idea.